demo/src/main/java/hu/flowacademy/stockmarket/persistance/model/Portfolio.java

```diff
@@ -0,0 +1,37 @@
+package hu.flowacademy.stockmarket.persistance.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "portfolio")
+public class Portfolio {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", updatable = false, nullable = false)
+    private Long id;
+
+    private String email;
+    private String stockSymbol;
+    private int amount;
+    private LocalDateTime buyingTime;
```

Comment:
Ha ez JSON-ből jön, akkor célszerű lenne egy `@JsonSerialize(using = LocalDateTimeSerializer.class)`-t használni

Reply:
Rendben akkor ezt használom ahogy kérted, meg is formáztam a dátumot, ami bekerül.